README.md

```diff
@@ -1,83 +1,126 @@
-    },
-    {
-        'title': "Hannnah Arendt's 'Human Condition' or How to Survive in a Men's World",
-        'abstract': "In this paper I want to analyze Hannah Arendt’s concepts, described in her Human Condition, from a perspective which takes into consideration her own fragile identity, placed in a particular way under the sign of the major influence of Martin Heidegger and, generally, under the influence of the men-politicians and men-philosophers. The triangle labor-work-action dissimulates an informal tendency to hide the woman’s condition under the human condition. The feminine and maternal spirit finds its expression here too, protesting against the child and childhood politicizing idea."
-    }
-]
-
-# load the model, point to the artifacts downloaded from s3
-data_dir = 'data/'
-s2ranker = S2FOS(data_dir)
-
-
-# get the predictions, which includes the following rules
-# If paper is English:
-#     If abstract exists:
-#         If any scores > -0.2:
-#             Take all predictions with score > -0.2
-#         Else:
-#             Take first prediction with score > -1.0
-#     If no abstract exists:
-#         Take first prediction with score > -0.2
-# Else:
-#     No predictions
-s2ranker.predict(papers)
-
-# also can get the raw decision scores without applingy any of the rules above
-s2ranker.decision_function(papers)
+from s2_fos import S2FOS
+
+papers = [{'title': "A Prototype-Based Few-Shot Named Entity Recognition",
+           'abstract': "Few-shot Named Entity Recognition (NER) task focuses on identifying name entities on a "
+                            "small amount of supervised training data. The work based on prototype network shows "
+                            "strong adaptability on the few-shot NER task. We think that the core idea of these "
+                            "approaches is to learn how to aggregate the representation of token mappings in vector "
+                            "space around entity class. But, as far as we know, no such work has been investigated its"
+                            " effect. So, we propose the ClusLoss and the ProEuroLoss aiming to enhance the model's "
+                            "ability in terms of aggregating semantic information spatially, thus helping the model "
+                            "better distinguish entity types. Experimental results show that ProEuroLoss achieves "
+                            "state-of-the-art performance on the average F1 scores for both 1-shot and 5-shot NER "
+                            "tasks, while the ClusLoss has competitive performance on such tasks.",
+            'journal_name': "Proceedings of the 8th International Conference on Computing and Artificial Intelligence",
```

Comment:
Do we always needs journal_name and venue_name? It's confusing to have both of these... Does it work fine if you JUST use one or the other? If so, let's change the example to only have one.